src/main/java/io/buildpacks/example/sample/CalculatorService.java

```diff
@@ -2,25 +2,82 @@
 
 import org.springframework.stereotype.Service;
 
```

Comment:
3. **Unit Testing**:
   - It is important to have corresponding unit tests for each method in the `CalculatorService` class to ensure correct functionality and handle edge cases under different scenarios.